jib-core/src/main/java/com/google/cloud/tools/jib/global/JibSystemProperties.java

```diff
@@ -91,6 +93,24 @@ public static boolean isUserAgentEnabled() {
     return Strings.isNullOrEmpty(System.getProperty(DISABLE_USER_AGENT));
   }
 
+  /**
+   * Gets whether to always cache base image layers. Determined from the {@code jib.cacheBaseImage}
+   * system property.
+   *
+   * @return true if the property is set to {@code always}; false otherwise
+   */
+  public static boolean alwaysCacheBaseImage() {
+    String property = System.getProperty(CACHE_BASE_IMAGE, "as-needed");
+    if ("always".equals(property)) {
```

Comment:
I'd rather we just make this a boolean property instead of a string needing two special keywords. I don't think this property in particular needs to be too future proof.

Reply:
Initially it was Boolean, but @briandealwis suggested these values. I thought this isn't a big deal to avoid being stuck with Boolean.

Reply:
I don't think `true`/`false` capture the situation, unfortunately.  `jib.cacheBaseImage=false` may still result in caching the base image with `dockerBuild` or `buildTar`.

Reply:
Then why not something like `jib.alwaysCacheBaseImage`?

Reply:
Sure that would work.

Reply:
Okay, I tried to think other potential modes but couldn't. I'll go back to Boolean.